keyboards/boardsource/equals/48/keymaps/via/keymap.c

```diff
@@ -0,0 +1,34 @@
+// Copyright 2022 @boardsource
+// SPDX-License-Identifier: GPL-2.0-or-later
+#include QMK_KEYBOARD_H
+
+enum layers {
+    _MAIN,
+    _RAISE,
+    _LOWER
+};
+
+#define LOWER MO(_LOWER)
+#define RAISE MO(_RAISE)
```

Comment:
Just a heads up, you may want to replace these with the tri layer keycodes: 
https://docs.qmk.fm/#/feature_tri_layer

VIA already supports these keycodes (as fn_mo13 and fn_mo23).